ui/src/main/webapp/src/app/shared/checkboxes.component.ts

```diff
@@ -24,168 +22,72 @@ export class CheckboxesComponent implements OnInit, OnChanges
      */
     @Input()
     valueCallback: (item: any) => string
-        = app => { throw new Error("valueCallback not yet defined." +
-                " This means Angular didn't keep the order of setting @Inputs."); };
+        = app => { throw new Error("valueCallback not yet defined.") };
```

Comment:
It should not happen, but the note was there because the order is not really guaranteed. Although I haven't seen that misbehave.